### ISSUE TITLE
Some ruins have invisible ? blocks.

### ISSUE THREAD

**Issue by VallenFrostweaver** [anegostudios/VintageStory-Issues#2900]
### Game Version

1.18.6

### Platform

Windows

### Modded

None

### SP/MP

None

### Description

I recently found some underground ruins (there was a translocator next to them as well) that had `Stone rubble` in it. The blocks themselves were invisible but for a block sized bounding box. When broken, it dropped the below image. I was on 1.18.6 and I don't find `Stone rubble` in the creative inventory list. Once I picked it up it turned into a ? block as seen below.
![image](https://github.com/anegostudios/VintageStory-Issues/assets/13642054/e70fd778-b58e-45e0-860f-c507faf77088)

**Comment by m-from-space** [comment]
I can confirm this and it is still happening in 1.18.8. I only encountered them two times and always as part of a "translocator" ruin.

**Comment by Bumber64** [comment]
Looks like the issue might be here in **blocktypes/clutter**:
```
randomizers: {
	"abacus": { types: ["abacus*"] },
	"bell": { types: ["bell*"] },
	"tank": { types: ["tank*"] },
	"weaponpile": { types: ["pile-weapon*"] },
	"machine": { types: ["pile-machine*"] },
	"stonerubble": { types: ["rubble-stone*"] },
	"stonerubble ": { types: ["rubble-wood*"] },
},
```

There's no such blocks as `rubble-stone*`. They're called `rubble-cracked*` for whatever reason, with shapes called "rubble-stone".

Also, what's the deal with the wood randomizer being called "stonerubble "?

**Comment by pingouyn** [comment]
Still a  bug in 1.19.0.rc6

**Comment by Th3Dilli** [comment]
should be fixed on latest 1.19.0-rc7/8
